src/app/containers/Timestamp/index.jsx

```diff
@@ -6,35 +6,37 @@ import relativeTime from './relativeTimestamp';
 // if the date is invalid return null - https://stackoverflow.com/questions/1353684/detecting-an-invalid-date-date-instance-in-javascript#answer-1353711
 const isValidDateTime = dateTime => !isNaN(dateTime); // eslint-disable-line no-restricted-globals
 
+const leadingZero = val => (val < 10 ? `0${val}` : val);
+
+const months = [ 'January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December' ]; // eslint-disable-line
```

Comment:
Can we add a `months` property to the service context config?

Reply:
IMO, as this PR is a bug fix, not a feature, that should be a separate issue, as this was previously hardcoded to `en-GB`

Happy to raise an issue/do it if you disagree though

Reply:
Fair point 👍 happy to leave it.